ParisBNF/abb/BNFabb203.xml

```diff
@@ -56,17 +56,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                              <explicit>
                              <locus target="#63ra"/>እምዝየ᎓ ምስለክሙ᎓ ወሞተ᎓ ዮሴፍ᎓
                              በ፻ወ፲ዓመት᎓ ወአምዓዝዎ᎓ ወሤምዎ᎓ በነፍቅ᎓ ውስተ᎓ ብሔረ᎓ ግብጽ።
+
                            </explicit>
                                </msItem>
 
                                <msItem xml:id="ms_i1.2">
                                  <locus from="1r"/>
                              <colophon xml:lang="gez" xml:id="coloph1">
```

Comment:
Comment on the `<colophon>`: It would of course be nice to mark up dates and the names of persons and places, if you have time (it seems very interesting!).

Reply:
addressed.